reader.c

```diff
@@ -151,6 +212,10 @@ int nsq_reader_connect_to_nsqd(struct NSQReader *rdr, const char *address, int p
     struct NSQDConnection *conn;
     int rc;
 
+    if (rdr->lookupd == NULL) {
+        rdr->address = strdup(address);
+        rdr->port = port;
+    }
```

Comment:
Yes, it's recommended to use `nsqlookupd` in that case, but it's still possible to connect to many specific `nsqd` - I don't think the library should preclude you from being able to do that.